### PR TITLE
Update: Integrate Directory with reports and dashboards

### PR DESCRIPTION
**Pull request by balajimanoharan**
Update: Integrate Directory with reports and dashboards

Tests need to be updated and PR merged, once the table links are in.

### CODE REVIEW

**Comment by mattgraham1995** [comment]
@balajimanoharan @jkusa The cyclical dependency is resolved

**Comment by balajimanoharan** [comment]
I cannot approve this, since I was the author of this PR. 👍 This should suffice. :shipit:

File: packages/directory/tests/dummy/app/routes/reports/new.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Route from '@ember/routing/route';
+import { inject } from '@ember/service';
+
+export default Route.extend({
```
Comment (jkusa) [inline packages/directory/tests/dummy/app/routes/reports/new.js]: is this in the dummy app?

File: packages/directory/tests/dummy/app/routes/reports.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Route from '@ember/routing/route';
+import { inject } from '@ember/service';
+
+export default Route.extend({
+  user: inject()
```
Comment (jkusa) [inline packages/directory/tests/dummy/app/routes/reports.js]: What is the `user` service used for?`

File: packages/directory/tests/dummy/app/routes/reports/new.js (hunk 1 above)
Comment (mattgraham1995) [reply packages/directory/tests/dummy/app/routes/reports/new.js]: Yes, when you create a new report from the directory, it will link here

**Comment by mattgraham1995** [comment]
@jkusa Tested against apps that do not use directory yet and there aren't any issues that I saw

File: packages/dashboards/package.json [hunk 3]
```diff
@@ -57,6 +57,7 @@
     "ember-font-awesome": "3.0.5",
     "ember-get-config": "0.2.4",
     "ember-gridstack": "1.2.0",
+    "ember-light-table": "^1.13.1",
```
Comment (jkusa) [inline packages/dashboards/package.json]: Do we need this now?

File: packages/reports/package.json [hunk 4]
```diff
@@ -60,6 +60,7 @@
     "ember-export-application-global": "^2.0.0",
     "ember-font-awesome": "3.0.5",
     "ember-get-config": "0.2.4",
+    "ember-light-table": "^1.13.1",
```
Comment (jkusa) [inline packages/reports/package.json]: Same question

File: packages/dashboards/addon/breakpoints.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export default {
```
Comment (jkusa) [inline packages/dashboards/addon/breakpoints.js]: Can we remove this?

**Pull request merged**